src/Eccube/Controller/ContactController.php

```diff
@@ -45,16 +43,6 @@ class ContactController extends AbstractController
      */
     protected $mailService;
 
-    /**
-     * @var EventDispatcherInterface
-     */
-    protected $eventDispatcher;
-
-    /**
-     * @var FormFactoryInterface
-     */
-    protected $formFactory;
-
     /**
      * @var AuthorizationCheckerInterface
```

Comment:
Use `$this->isGranted('ROLE_XXX')` instead of `$this->authorizationChecker->isGranted('ROLE_USER')`